src/Tizen.NUI/src/public/XamlBinding/BindableObject.cs

```diff
@@ -36,14 +36,39 @@ public abstract class BindableObject : INotifyPropertyChanged, IDynamicResourceH
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static readonly BindableProperty BindingContextProperty =
-            BindableProperty.Create(nameof(BindingContext), typeof(object), typeof(BindableObject), default(object),
-                                    BindingMode.OneWay, null, BindingContextPropertyChanged, null, null, BindingContextPropertyBindingChanging);
+            BindableProperty.Create(nameof(BindingContext), typeof(object), typeof(BindableObject), null, propertyChanged: (BindableProperty.BindingPropertyChangedDelegate)((bindable, oldValue, newValue) =>
```

Comment:
could you check this?

https://github.com/Samsung/TizenFX/pull/3122#discussion_r642180063

thank you.